files/orchestrate/src/create-faucet/app.ts

```diff
@@ -1,25 +1,25 @@
 // tslint:disable: no-console
 
-import { FaucetRegistry } from 'pegasys-orchestrate'
+import { OrchestrateClient } from "pegasys-orchestrate";
 
 export const start = async () => {
   try {
-    const faucetRegistry = new FaucetRegistry(process.env.CHAIN_REGISTRY_HOST!)
+    const faucetRegistry = new OrchestrateClient(process.env.API_HOST!);
 
     const faucet = await faucetRegistry.registerFaucet(
       {
         name: `${process.env.CHAIN}-faucet`,
         chainRule: process.env.CHAIN_UUID!,
         creditorAccount: process.env.FAUCET_ACCOUNT!,
```

Comment:
Per my other comment - might want to add a check on this env var in this script with a user-friendly error message, as many users will either not read the README, or will skim it.

Reply:
The error won't be confusing but will be something of the sort: `Invalid From account` or `Invalid creditor address`. Given that the `.env` must be filled and that is explained in the README, I see no problem with this. Also, it would have failed as well before the changes since these hardcoded accounts don't exist and are invalid anyways.